### PR TITLE
Fix inputs to hl2ss_ros_bridge based on updates to that node

### PR DESCRIPTION
**Pull request by hdefazio**

### CODE REVIEW

**Comment by hdefazio** [comment]
@cameron-a-johnson  / @Purg

File: tmux/record/record_ros_bag.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ root: <%= ENV["ANGEL_WORKSPACE_DIR"] %>
 # on_project_start: command
 on_project_start: |
   export ROS_NAMESPACE=${ROS_NAMESPACE:-/kitware}
-  export HL2_IP=${HL2_IP:-192.168.1.101}
+  export HL2_IP=${HL2_IP:-192.168.1.4}
```
Comment (Purg) [inline tmux/record/record_ros_bag.yml:22]: Why is the IP of the Hololens not '.101'?

Comment (hdefazio) [reply tmux/record/record_ros_bag.yml:22]: Here at KRS it is .4 on the wifi.... I can keep the default .101 but maybe it's better for the default to be what we use on Remy?

Comment (Purg) [reply tmux/record/record_ros_bag.yml:22]: sure, leave it at 4 then.

**Review by Purg — APPROVED** [review]

**Pull request merged**